engine/security/rootless.md

```diff
@@ -5,30 +5,29 @@ title: Run the Docker daemon as a non-root user (Rootless mode)
 ---
 
 Rootless mode allows running the Docker daemon and containers as a non-root
-user, for the sake of mitigating potential vulnerabilities in the daemon and
+user to mitigate potential vulnerabilities in the daemon and
 the container runtime.
 
-Rootless mode does not require root privileges even for installation of the
-Docker daemon, as long as [the prerequisites](#prerequisites) are satisfied.
+Rootless mode does not require root privileges even during the installation of the Docker daemon, as long as the [prerequisites](#prerequisites) are met.
```

Comment:
could you keep this wrapped to ~80-chars?

```suggestion
Rootless mode does not require root privileges even during the installation of
the Docker daemon, as long as the [prerequisites](#prerequisites) are met.
```

Reply:
I have this setting on my VS Code and the content looks wrapped to 80 chars, not sure why didn't honor the setting when I pushed the updates.